certified-connectors/Fliplet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,408 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Fliplet REST APIs",
+    "description": "This definition contains a list of action on Fliplet data sources",
+    "contact": {
+      "name": "Fliplet",
+      "url": "https://help.fliplet.com/",
+      "email": "support@fliplet.com"
+    },
+    "version": "0.2"
+  },
+  "host": "api.fliplet.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/apps": {
+      "get": {
+        "summary": "Gets a list of apps for this API token",
+        "description": "Get all apps",
+        "operationId": "getAllApps",
```

Comment:
@squallstar all operationIds should be in Camel case. please change to "GetAllApps". Please fix all operationIds in the swagger file.